src/template/page.ejs

```diff
@@ -6,22 +6,27 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title><%= title %></title>
-    <link rel="stylesheet" href="<%- asset.bootstrap %>">
-    <link rel="stylesheet" href="<%- asset.bootstrapVue %>">
-    <link rel="stylesheet" href="<%- asset.fontAwesome %>" >
-    <link rel="stylesheet" href="<%- asset.glyphicons %>" >
-    <link rel="stylesheet" href="<%- asset.highlight %>">
```

Comment:
These should not be moved to the bottom:
1. The convention is to put stylesheets at the head.
2. Authors may need to override the stylesheets, and they currently do that by adding additional stylesheets in head. Moving it to the bottom prevents them from overriding the stylesheets.

Reply:
The reason I moved them to the bottom is to reduce the time taken to start showing the spinner. Otherwise, we have to wait for the other much larger assets to load before the spinner is shown. On slower connections this delay can be several seconds. 

If this issue is not a concern, I will remove these changes!

Reply:
The main bottleneck of the loading process is vue-strap and bulky/embedded page content and assets (e.g. youtube videos). Stylesheets themselves are fine and do not usually take up most of the loading time.

Reply:
Will remove these changes!